app/lib/dartdoc/dartdoc_runner.dart

```diff
@@ -257,7 +269,7 @@ class DartdocJobProcessor extends JobProcessor {
               '$undocumented out of $total API elements (library, class, field '
               'or method) have no adequate dartdoc content. Good documentation '
               'improves code readability and discoverability through search.',
```

Comment:
We should perhaps update this comment to explain the non-linear scores

Reply:
ack